Modules/IO/PNG/include/itkPNGImageIO.h

```diff
@@ -59,6 +59,9 @@ class ITKIOPNG_EXPORT PNGImageIO : public ImageIOBase
    * 0 is returned */
   itkGetConstReferenceMacro(ColorPalette, PaletteType);
 
+  /** Set the palette of the image. */
+  itkSetMacro(ColorPalette, PaletteType);
```

Comment:
[SetMacro](https://github.com/InsightSoftwareConsortium/ITK/blob/2074c2f9e5ed3f087d0e2059f8e0e8992fcad7ef/Modules/Core/Common/include/itkMacro.h#L873) calls [Debug macro](https://github.com/InsightSoftwareConsortium/ITK/blob/2074c2f9e5ed3f087d0e2059f8e0e8992fcad7ef/Modules/Core/Common/include/itkMacro.h#L396), which attempts to `ostream<<x`. This is not defined for `std::vector`.

Reply:
Sorry, I didn't got any compilation error on my machine. How should I fix this ? Not using the macro ?

Reply:
Yes, make a code similar to the macro. Maybe copy-paste and remove the debug output statement?